### ISSUE TITLE
Script to manage failover-failback

### ISSUE THREAD

**Issue by Tusamarco** [Tusamarco/proxy_sql_tools#12]
From @altmanmarcelo
It will be useful to have the script identify when a Node comes back and is then able to take back is Writer role (if it has higher priority, in group 8000 or PXC_CLUSTER_VIEW).
Case:
Node1 - Weight 1000
Node2 - Weight 100
Node3 - Weight 10

Node 1 crash -> traffic to Node2
Node1 recover -> NEW traffic Node2 | Old traffic (existing connections) terminate on Node2

Marcelo can you please confirm this is what you were asking in the chat?

**Comment by altmannmarcelo** [comment]
@Tusamarco yes, that is the idea of failover back. Once node1 is back and with the Synced state, if it has a higher priority over the current writer, failover back should occur.

**Comment by Tusamarco** [comment]
@altmannmarcelo check #14 is that relevant for you too?
